### DISCUSSION TITLE
novelWriter has been added to deb-get

### DISCUSSION THREAD

**Discussion by sbstn87** [vkbo/novelWriter#1871] [General]
Just FYI, novelWriter has been recently added to deb-get:
https://github.com/wimpysworld/deb-get/blob/main/01-main/README.md

What is deb-get:
deb-get makes it easy to install and update .debs published in 3rd party apt repositories or made available via direct download on websites or GitHub release pages.

**Comment by vkbo** [comment]
Ah, cool 😃

**Reply by vkbo** [reply]
That said, I wish people wouldn't call it a "markdown editor" because it really isn't. It uses only a subset of markdown, pluss other markup styles.
